docs/dev/table/connectors/kafka.md

```diff
@@ -100,11 +100,18 @@ Connector Options
     </tr>
     <tr>
       <td><h5>topic</h5></td>
-      <td>required</td>
+      <td>required for sink, optional for source(use 'topic-pattern' instead if not set)</td>
       <td style="word-wrap: break-word;">(none)</td>
       <td>String</td>
-      <td>Topic name from which the table is read.</td>
+      <td>Topic name(s) from which the table is read. It also supports topic list for source by separating topic by comma like <code>'topic-1, topic-2'</code>.</td>
     </tr>
+    <tr>
+      <td><h5>topic-pattern</h5></td>
+      <td>optional for source(use 'topic' instead if not set)</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Topic pattern from which the table is read. It will use input value to build regex expression to discover matched topics.</td>
```

Comment:
The regular expression for a pattern of topic names to read from. All topics with names that match the specified regular expression will be subscribed by the consumer when the job starts running. Note, only one of "topic-pattern" and "topic" can be specified for sources.